examples/remoteBuild.md

```diff
@@ -0,0 +1,69 @@
+# remote targets
+
+`kustomize` supports the best practice of storing one's
+entire configuration in a version control system such as Github.
+With configuration and kustomization.yaml available remotely,
+the url can be treated as a remote target. Users can run `kustomize build`
+against it or list it as `bases` in an overlay.
+
+
+The url should follow
+[hashicorp/go-getter URL format](https://github.com/hashicorp/go-getter#url-format)
+Here are some example urls pointing to Github repos following this convention.
+
+- a repo with a root level kustomization.yaml
+
+  `github.com/Liujingfang1/mysql`
+- a repo with a root level kustomization.yaml on branch test
+
+  `github.com/Liujingfang1/mysql?ref=test`
+- a subdirectory in a repo on version v1.0.6
+
+  `github.com/kubernetes-sigs/kustomize//examples/multibases?ref=v1.0.6`
+- a subdirectory in a repo on branch repoUrl2
+
+  `github.com/Liujingfang1/kustomize//examples/helloWorld?ref=repoUrl2`
+- a subdirectory in a repo on commit `7050a45134e9848fca214ad7e7007e96e5042c03`
+
+  `github.com/Liujingfang1/kustomize//examples/helloWorld?ref=7050a45134e9848fca214ad7e7007e96e5042c03`
+
+
+## kustomize build {URL}
+
+`kustomize build` can be run against a url. The effect is the same as cloing the repo, checking out the specified ref,
```

Comment:
I think the entire doc here should start with this sentence.

The current first section offers a list of example URLs.  That would be better at the end.